File: C_Programming/Assignment/Assignment2/marks.c
#include<stdio.h>
void main()
{
    int marks;
    printf("Enter the marks:");
    scanf("%d",&marks);
    if(marks>75)
    {
        printf("Distinction pass");
    }
    else if(marks>65)
    {
        printf("First Class pass");
    }
    else if(marks>55)
    {
        printf("Second class pass");
    }
    else if(marks>40)
    {
        printf("Pass class");
    }
    else if(marks<40)
    {
        printf("fail");
    }
    else
    {
       printf("Enter the valid  marks");
    }
}

//output:Enter the marks:78
//Distinction pass
// Enter the marks:34
// fail

File: Java_Programs/Assignment2/Marks.java
package Assignment2;

public class Marks {

	public static void main(String[] args) {
		int marks=98;
//	    printf("Enter the marks:");
//	    scanf("%d",&marks);
	    if(marks>75)
	    {
	        System.out.printf("Distinction pass");
	    }
	    else if(marks>65)
	    {
	    	System.out.printf("First Class pass");
	    }
	    else if(marks>55)
	    {
	    	System.out.printf("Second class pass");
	    }
	    else if(marks>40)
	    {
	    	System.out.printf("Pass class");
	    }
	    else if(marks<40)
	    {
	    	System.out.printf("fail");
	    }
	    else
	    {
	    	System.out.printf("Enter the valid  marks");
	    }

	}

}
